concepts/booleans/introduction.md

```diff
@@ -1,22 +1,55 @@
 # Introduction
 
-Booleans in Rust are represented by the `bool` type, which values can be either `true` or `false`.
+## Booleans
 
-Rust supports six comparison operators: `==` (Equal), `!=` (Not equal), `>` (Greater than), `<` (Less than), `>=`
-(Greater than or equal to), `<=` (Less than or equal to). They can be used to evaluate the relationship of non-boolean
-values into a boolean value.
+As in most other programming languages, a Boolean type in Rust has two possible values: `true` and `false`.
+
+### Declaration
+The Boolean type in Rust is specified using `bool`.
+
+```rust
+let t = true;
+
+let f: bool = false; // with explicit type annotation
+```
+
+The main way to use Boolean values is through conditionals, such as an if
```

Comment:
What is the exact purpose of `about.md` and `introduction.md` on a concept? There is a lot of duplicated text between the two files, is that normal? I guess would be that some exaplantion should go in one of the two files - not in both.

@ErikSchierboom

Reply:
These are documented here: https://exercism.org/docs/building/tracks/concepts

`introduction.md` is shown to students who have not yet solved the concept exercise(s?), and `about.md` is shown to those who have.

It is entirely normal for the two to share a lot of text. It also happens that they are identical.

Reply:
Yep, that's entirely correct.